content/5_Plat/Sweep_Line.mdx

```diff
@@ -162,8 +162,162 @@ For each point, we recalculate $\delta = \min(\delta, \delta_p)$, and update our
 <FocusProblem problem="segSam" />
 
 ### Solution
+Let's simplify the problem a little bit and focus on finding any overlapping segments. The segment we are looking for is one of the two.
```

Comment:
> The segment we are looking for is one of the two.

what

Reply:
they probably mean "one of the two segments"... but yes, please clarify this

Reply:
Ok it was a mistake